WordPress/src/main/java/org/wordpress/android/ui/posts/EditPostActivity.java

```diff
@@ -528,10 +528,8 @@ protected void onCreate(Bundle savedInstanceState) {
         // ok now we are sure to have both a valid Post and showGutenberg flag, let's start the editing session tracker
         createPostEditorAnalyticsSessionTracker(mShowGutenbergEditor, mPost, mSite);
 
-        if (savedInstanceState == null) {
-            // Bump the stat the first time the editor is opened.
-            PostUtils.trackOpenPostAnalytics(mPost, mSite);
-        }
+        // Bump editor opened event every time the activity is created, to match the EDITOR_CLOSED event onDestroy
```

Comment:
For the sake of matching events and meaning then (assuming we want to go along the lines of [my previous comment](https://github.com/wordpress-mobile/WordPress-Android/pull/9901#issuecomment-494505791)), what about issuing the `EDITOR_OPEN` and `EDITOR_CLOSED` events in `onResume()` and `onStop()` lifecycle methods? As these would seem to reflect the editor being shown on screen / taken to the background respectively and are guaranteed to be called ([as opposed to `onDestroy`](https://developer.android.com/reference/android/app/Activity.html#onDestroy())) when that happens. What do you think?

Reply:
Yes, I agree and will make the changes.  

Anyway, i'm not still fully convinced we need to go along this way, since don't see much meaning on tracking the number of "impressions" the editor has had on the screen. 
Those 2 events were there - i guess - to track the number of times the editor was opened, but then we noticed there were problems tracking it, and then introduced the concept of session. 
I guess we can get rid of those events, maybe in another PR. 🤷‍♂ 

Will update the event descriptions in tracks and the spreadsheet once this land as @aerych suggested.

Reply:
Wondering whether it's possible to add a comment to the descriptions in Tracks, saying that the events have been "repurposed" in a way since approximately day "MM/DD/YYYY"? Maybe a link to this discussion or something alike, so people in the future won't be misguided in their interpretation of data? wdyt?

Reply:
> since don't see much meaning on tracking the number of "impressions" the editor has had on the screen.

BTW forgot to mention I agree on this  as well, and since we already have the concept of session as you mentioned I feel it's ok to just correct the `opened` and `closed` to be less of an interpretation and just _that_ :) 👍